selfdrive/controls/tests/test_monitoring.py

```diff
@@ -87,23 +87,24 @@ def test_rhd_parser(self):
 
   # 0. op engaged, driver is doing fine all the time
   def test_fully_aware_driver(self):
-    events_output = run_DState_seq(always_attentive, always_false, always_true, always_false)
+    events_output = run_DState_seq(always_attentive, always_false, always_true, always_false)[0]
     self.assertTrue(np.sum([len(event) for event in events_output])==0)
 
   # 1. op engaged, driver is distracted and does nothing
   def test_fully_distracted_driver(self):
-    events_output = run_DState_seq(always_distracted, always_false, always_true, always_false)
+    events_output, d_status = run_DState_seq(always_distracted, always_false, always_true, always_false)
     self.assertTrue(len(events_output[int((_DISTRACTED_TIME-_DISTRACTED_PRE_TIME_TILL_TERMINAL)/2/DT_DMON)])==0)
     self.assertEqual(events_output[int((_DISTRACTED_TIME-_DISTRACTED_PRE_TIME_TILL_TERMINAL+\
                       ((_DISTRACTED_PRE_TIME_TILL_TERMINAL-_DISTRACTED_PROMPT_TIME_TILL_TERMINAL)/2))/DT_DMON)][0].name, 'preDriverDistracted')
     self.assertEqual(events_output[int((_DISTRACTED_TIME-_DISTRACTED_PROMPT_TIME_TILL_TERMINAL+\
                       ((_DISTRACTED_PROMPT_TIME_TILL_TERMINAL)/2))/DT_DMON)][0].name, 'promptDriverDistracted')
     self.assertEqual(events_output[int((_DISTRACTED_TIME+\
                       ((_TEST_TIMESPAN-10-_DISTRACTED_TIME)/2))/DT_DMON)][0].name, 'driverDistracted')
+    self.assertTrue(type(d_status.awareness)==float)
```

Comment:
Use `self.assertNotIsInstance(d_status.awareness, np.float64)`

Reply:
Unfortunately `np.float64` is a inherited from `float`, so you can't user `isinstance(x, float)`